lib/entities/console.rb

```diff
@@ -14,105 +15,147 @@
          "describe_diff": 'difficult', "user_answer": 'user_answer',
          "wrong_choice": 'wrong_choice', "name": 'name',
          "win": 'win', "failure": 'failure',
-         "restart?": 'restart?', "save?": 'save?' }.freeze
+         "restart?": 'restart?', "save?": 'save?',
+         "statistics": 'statistics', "game_attemt": 'game_attemt',
+         "game_hint": 'game_hint', "continue?": 'continue?' }.freeze
 
 class Console
-  include Storage
-  include Validation
+  include Load
+  include Validate
 
   def initialize(send_to_console = 'greeting')
-    print I18n.t(send_to_console).colorize(:light_blue)
+    print I18n.t(send_to_console)
   end
 
-  def question
-    print yield.colorize(:light_green)
-    gets.chomp
+  def answer_for_user(answer)
+    puts answer
   end
 
-  def what_next?
-    question { '' }
+  def question
+    print yield
+    gets.chomp
   end
 
-  def answer_for_user(answer)
-    puts "Secret code has #{answer}"
-  end
-
-  def choice(user_choice)
-    case user_choice
-    when MENU[:no] then goodbye
-    when MENU[:yes] then choose_the_command
-    when MENU[:game_rules] then rules
-    when MENU[:stats] then stats
-    when MENU[:game_start] then start
-    when MENU[:exit] then goodbye
-    else
-      puts I18n.t(MENU[:wrong_choice]).colorize(:red)
-      choose_the_command
+  def game_over(s_code, _game_statistics, game_status = 'failure')
+    puts "Secret code is #{s_code.join}"
+    case game_status
+    when 'win' then puts I18n.t(MENU[:win])
+    when 'failure' then puts I18n.t(MENU[:failure])
     end
+    save?(_game_statistics)
+    start?
+    goodbye
   end
 
-  def choose_the_command
-    puts I18n.t(MENU[:choose_the_command]).colorize(:light_yellow)
-    choice(question { I18n.t(MENU[:user_answer]) })
+  def goodbye
+    puts 'exit'.chomp
   end
 
-  def goodbye
-    puts I18n.t(MENU[:goodbye]).colorize(:light_yellow)
-    exit
+  def start
+    Console_game.new(name, difficulty).game_progress
   end
 
-  def exit
-    puts 'exit'.chomp
+  def choice
+    puts I18n.t(MENU[:continue?])
+    loop do
+      case question {}
+      when MENU[:yes] then
+        puts I18n.t(MENU[:choose_the_command])
+        next
+      when MENU[:no]
+        goodbye
+        break
+      when MENU[:exit]
+        goodbye
+        break
+      when MENU[:game_rules] then return rules
+      when MENU[:stats] then return stats
+      when MENU[:game_start] then start
+      else
+        puts I18n.t(MENU[:wrong_choice])
+        next
+      end
+    end
   end
 
+  private
+
   def rules
-    puts I18n.t(MENU[:game_rules]).colorize(:light_cyan)
-    choice(what_next?)
+    puts I18n.t(MENU[:game_rules])
+    choice
+  end
+
+  def print_statistic
+    load_statistics(FILE_NAME_ST).each_with_index do |value, index|
+      puts I18n.t(MENU[:statistics], rating: index + 1, name: value[:user_name], difficulty: value[:difficulty],
```

Comment:
I think you can pass a value object to I18n instead of passing `name`, `difficulty` and so on